From khali@linux-fr.org Fri Oct  7 15:15:18 2005
Date: Sat, 8 Oct 2005 00:14:17 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 11/16] i2c: Drop unused per-i2c-algorithm adapter max
Message-Id: <20051008001417.5efb1573.khali@linux-fr.org>

There are no more per-i2c-algorithm adapter max. Last time there were
was in July 1999.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 include/linux/i2c-algo-bit.h |    2 --
 include/linux/i2c-algo-pca.h |    2 --
 include/linux/i2c-algo-pcf.h |    2 --
 3 files changed, 6 deletions(-)

--- gregkh-2.6.orig/include/linux/i2c-algo-bit.h
+++ gregkh-2.6/include/linux/i2c-algo-bit.h
@@ -44,8 +44,6 @@ struct i2c_algo_bit_data {
 	int timeout;		/* in jiffies */
 };
 
-#define I2C_BIT_ADAP_MAX	16
-
 int i2c_bit_add_bus(struct i2c_adapter *);
 int i2c_bit_del_bus(struct i2c_adapter *);
 
--- gregkh-2.6.orig/include/linux/i2c-algo-pca.h
+++ gregkh-2.6/include/linux/i2c-algo-pca.h
@@ -9,8 +9,6 @@ struct i2c_algo_pca_data {
 	int  (*wait_for_interrupt)	(struct i2c_algo_pca_data *adap);
 };
 
-#define I2C_PCA_ADAP_MAX	16
-
 int i2c_pca_add_bus(struct i2c_adapter *);
 int i2c_pca_del_bus(struct i2c_adapter *);
 
--- gregkh-2.6.orig/include/linux/i2c-algo-pcf.h
+++ gregkh-2.6/include/linux/i2c-algo-pcf.h
@@ -39,8 +39,6 @@ struct i2c_algo_pcf_data {
 	int timeout;
 };
 
-#define I2C_PCF_ADAP_MAX	16
-
 int i2c_pcf_add_bus(struct i2c_adapter *);
 int i2c_pcf_del_bus(struct i2c_adapter *);
 
